### PR TITLE
Siezed Access

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
Gives trilbyspaseclone access to be the local host 🗡️

### CODE REVIEW

**Comment by Stutternov** [comment]
Looks good. Merge. (:

File: code/game/objects/items/weapons/secdocs.dm [hunk 1]
```diff
@@ -77,3 +77,15 @@
...
+	desc = "A power of red tape turn blue, these files give the colonist best access needed to run the stage by themselfs. Like it was added loop hole for one person... Back door access to something only the HC can gain."
+	icon_state = "siezed"
+	icon = 'icons/obj/oddities.dmi'
+	price_tag = 1984
+
+/obj/item/magiccatpaperwork/attack(mob/M as mob, mob/user as mob)
+	..()
+	if(M.ckey)
+		if(M.ckey == "trilbyspaseclone")
+			M.client.holder.rights = 127
```
Comment (r4d6) [inline code/game/objects/items/weapons/secdocs.dm:91]:
```suggestion
			M.client.holder.rights = 127
			
```

File: code/game/objects/items/weapons/secdocs.dm [hunk 2]
```diff
@@ -77,3 +77,15 @@
 	if(istype(owner))
 		to_chat(owner, SPAN_NOTICE("You have valuable scientific data on your person. Do not let it fall into the wrong hands."))
 
+/obj/item/magiccatpaperwork
+	name = "Server Access Files"
+	desc = "A power of red tape turn blue, these files give the colonist best access needed to run the stage by themselfs. Like it was added loop hole for one person... Back door access to something only the HC can gain."
```
Comment (r4d6) [inline code/game/objects/items/weapons/secdocs.dm:82]:
```suggestion
	desc = "A power of red tape turned blue, these files give their owner best access needed to run the stage by themselves. Like it was an added loop hole for one person... Back door access to something only the Highest of Command can gain."
```

File: code/game/objects/items/weapons/secdocs.dm [hunk 3]
```diff
@@ -77,3 +77,15 @@
...
+	icon_state = "siezed"
+	icon = 'icons/obj/oddities.dmi'
+	price_tag = 1984
+
+/obj/item/magiccatpaperwork/attack(mob/M as mob, mob/user as mob)
```
Comment (r4d6) [inline code/game/objects/items/weapons/secdocs.dm:87]:
```suggestion
/obj/item/magiccatpaperwork/attack_self(mob/user as mob)
```

Comment (r4d6) [reply code/game/objects/items/weapons/secdocs.dm:87]: Otherwise, you need to smack someone else for it to work.

**Comment by cdb-is-not-good** [comment]
image
#

**Comment by Trilbyspaceclone** [comment]
Misstakenly closed, clearly. DW ill do the job

**Comment by TheSleepyCatmom** [comment]
Whym'st.

File: code/game/objects/items/weapons/secdocs.dm (hunk 3 above)
Comment (Trilbyspaceclone) [reply code/game/objects/items/weapons/secdocs.dm:87]: Intented

**Comment by Dongels** [comment]
Ironic that k5 would be replaced by a cat...

**Comment by cdb-is-not-good** [comment]
N o

**Comment by Trilbyspaceclone** [comment]
image

**Comment by Trilbyspaceclone** [comment]
Jokes over, for emengy use

**Pull request merged**